api/src/main/java/org/apache/iceberg/view/VersionBuilder.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.view;
+
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.catalog.Namespace;
+
+public interface VersionBuilder<T> {
+  /**
+   * Set the view schema.
+   *
+   * @param schema The schema to use for this view version
+   * @return this for method chaining
+   */
+  T withSchema(Schema schema);
+
+  /**
+   * Add a view representation for the given dialect and the SQL to the view.
+   *
+   * @param dialect The dialect of the view representation
+   * @param sql The SQL of the view representation
+   * @return this for method chaining
+   */
+  T withQuery(String dialect, String sql);
+
+  /**
+   * Add a view representation for the given dialect and the SQL to the view.
```

Comment:
This Javadoc isn't correct. Looks like a copy/paste problem.

Reply:
thanks for catching this, fixed